package.json

```diff
@@ -1,1622 +1,1663 @@
 {
-  "name": "vscode-openshift-connector",
```

Comment:
Please, return original tab size back. It is hard to see what was actually changed.

Reply:
Done.